src/util/types.ts

```diff
@@ -11,9 +11,4 @@ export interface RSPType {
 export interface RSPServer {
     type: RSPType;
     state: number;
-}
-
-export interface ServerInfo {
-    host: string;
-    port: number;
```

Comment:
ServerInfo is a type used by external RSP Provider so we should keep it here. We prefer to keep this type in the API so external contributors can start creating their own extensions with just this npm package and anything else.

Reply:
the interface was declared twice, so I deleted the duplicate